xml/security_cryptofs.xml

```diff
@@ -370,36 +363,49 @@
+    <xref linkend="sec.security.cryptofs.y2.vdisk"/>.
    </para>
   </warning>
  </sect1>
- <sect1 xml:id="sec.security.cryptofs.vi">
-  <title>Using vi to Encrypt Single ASCII Text Files</title><indexterm>
-
-  <primary>encrypting</primary>
-
-  <secondary>files with vi</secondary></indexterm>
+ <sect1 xml:id="sec.security.cryptofs.gpg">
+  <title>Encrypting Files with GPG</title>
 
   <para>
-   The disadvantage of using encrypted partitions is obvious: While the
-   partition is mounted, at least
-   <systemitem class="username">root</systemitem> can access the data. To
-   prevent this, <command>vi</command> can be used in encrypted mode.
+   The GPG encryption software can be used to encrypt individual files and documents.
   </para>
 
   <para>
-   Use <command>vi</command> <option>-x</option>
-   <replaceable>FILENAME</replaceable> to edit a new file.
-   <command>vi</command> prompts you to set a password, after which it
-   encrypts the content of the file. Whenever you access this file,
-   <command>vi</command> requests the correct password.
+   To encrypt a file with GPG, you need to generate a key pair first. To do
+   this, run the <command>gpg --gen-key</command> and follow the on-screen
+   instructions. When generating the key pair, GPG creates a user ID (UID) to
+   identify the key based on your real name, comments, and email address. You
+   need this UID (or just a part of it like your first name or email address)
+   to specify the key you want to use to encrypt a file. To find the UID of an
+   existing key, use the <command>gpg --list-keys</command> command. To encrypt
+   a file use the following command:
   </para>
-
+  <screen>gpg -e -r <replaceable>UID</replaceable>
+  <replaceable>FILE</replaceable></screen>
+  <para>
+    Replace <replaceable>UID</replaceable> with part of the UID (for example,
+    your first name) and <replaceable>FILE</replaceable> with the file you want
+    to encrypt. For example:
+  </para>
+  <screen>gpg -e -r Tux secret.txt</screen>
   <para>
-   For even more security, you can place the encrypted text file in an
-   encrypted partition. This is recommended because the encryption used in
-   <command>vi</command> is not very strong.
-  </para><indexterm startref="idx.encrypting.files" class="endofrange"/><indexterm startref="idx.encrypting" class="endofrange"/>
+    This command creates an encrypted version of the specified file
+    recognizable by the <filename>.gpg</filename> file extension (in
+    this example, it is <filename>secret.txt.gpg</filename>).
+  </para>
+  <para>
+    To decrypt an encrypted file, use the following command:
+  </para>
+  <screen>gpg -d -o <replaceable>DECRYPTED_FILE</replaceable>
+  <replaceable>ENCRYPTED_FILE</replaceable></screen>
+  <para>
+    Replace <replaceable>DECRYPTED_FILE</replaceable> with the desired name for
+    the decrypted file and <replaceable>ENCRYPTED_FILE</replaceable> with the
+    encrypted file you want to decrypt.
+  </para>
```

Comment:
You should mention, that a file encrypted this way can _only_ be decrypted with the private key of the same user that has encrypted it. To share an encrypted file with a second person, you additionally need that person's public key for encryption.